msteams-platform/sbs-graphactivity-feedbroadcast.yml

```diff
@@ -0,0 +1,381 @@
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/manifestfile.png" alt-text="Manifest file" border="true":::
+
+- title: Build and run the service
+  durationInMinutes: 1
+  content: |
+    **To build and run the service using Visual Studio
+
+    # [Latest version of Visual Studio](#tab/vs2022)
+
+       1. Open the latest version of Visual Studio.
+       1. Navigate to **File** > **Open** > **Project/Solution**.
+    
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/VSopenfile.png" alt-text="Open file in Visual Studio" border="true":::
+
+       1. Select **ActivityFeedBroadcast.sln** file from **csharp** folder.
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/openproject.png" alt-text="Open a project" border="true":::
+
+       1. Press **F5** to run the project.
+    
+       1. Select **Yes** if the following dialog appears:
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/graphactivity-generator-certificate.png" alt-text="Select certificate" border="true":::
+
+          A webpage opens with a message **Your bot is ready!**
+
+          :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/appisready.png" alt-text="App is ready" border="true":::
+
+   
+- title: Add activity feed broadcast app to Teams
+  durationInMinutes: 1
+  content: |
+    **To add activity feed broadcast app to Teams**
+
+             
+    1. In your cloned repository, navigate to **csharp > ActivityFeedBroadcast > AppPackage**.
+
+    1. Create a .zip with the following files that are present in the **AppPackage** folder: 
+       * manifest.json
+       * icon-outline.png
+       * icon-color.png
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/selectzip.png" alt-text="Select zip file" border="true":::
+    
+    1. Go to **Microsoft Teams**.
+
+    1. Select **Apps**.
+
+    1. Select **Manage your apps**.
+    
+    1. Select **Upload a custom app**. 
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/openacustomapp.png" alt-text="Upload a custom app" border="true":::
+
+    1. Select **Open** to upload the .zip file that you created in the **AppPackage** folder.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/openacustomapp.png" alt-text="Open a custom app" border="true":::
+
+    1. Select **Add**.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/addanapp.png" alt-text="Add app" border="true":::
+   
+          The app is visible in the meeting SidePanel. 
+
+    1. The **Broadcast Tab** displays Message title and Message description.
+
+        :::image type="content" source="~/assets/images/sbs-graphactivity-feedbroadcast/output1.png" alt-text="Output" border="true":::
```

Comment:
This image is not required. You can add the steps:

1. The **Broadcast Tab** displays Message title and Message description.
2. Enter the message and select **Send message.**

Update the image: output2.png with correct highlights, please use red box to highlight UI.